ONSdigital/respondent-home-ui
# Access code accessibility

### What is the context of this PR?

Fixes #7, #8 and #9 

### How to review

- Check the access code has more descriptive labels.
- Check access code uses font-family: monaco, Consolas, "Lucida Console", monospace. 
- Check access code on mobile and other browsers
- Check inputs have the border colour: #999999
